app/views/documents/show/_summary_tab.html.erb

```diff
@@ -20,6 +20,14 @@
     <% end %>
 
     <%= render "documents/show/tags" %>
+
+    <%
+      # TODO: Once Access Limiting has been implemented, this logic should only
```

Comment:
I don't think we need this here, since we will certainly change this when the feature is built, and the specs protect us from breaking it for backdating.